src/main/webapp/index-script.js

```diff
@@ -303,6 +302,24 @@ function handleSearchResults(results, service) {
   });
 }
 
+/** Deletes all markers in the array by removing references to them. */
+function deleteMarkers() {
+  clearMarkers();
+  markers = [];
+}
+
+/** Removes the markers from the map, but keeps them in the array. */
+function clearMarkers() {
```

Comment:
It seems you don't need this function, why not just use the other function or call it clearMarkers?